Kms/src/V1/KeyManagementServiceClient.php

```diff
@@ -18,37 +18,17 @@
 /*
  * GENERATED CODE WARNING
  * Generated by gapic-generator-php from the file
- * https://github.com/google/googleapis/blob/master/google/cloud/kms/v1/service.proto
+ * https://github.com/googleapis/googleapis/blob/master/google/cloud/kms/v1/service.proto
  * Updates to the above are reflected here through a refresh process.
  */
 
 namespace Google\Cloud\Kms\V1;
 
 use Google\Cloud\Kms\V1\Gapic\KeyManagementServiceGapicClient;
-use Google\ApiCore\PathTemplate;
 
 /** {@inheritdoc} */
 class KeyManagementServiceClient extends KeyManagementServiceGapicClient
 {
-    /**
-     * Formats a string containing the fully-qualified path to represent
-     * a crypto_key_path resource.
-     *
-     * @param string $project
-     * @param string $location
-     * @param string $keyRing
-     * @param string $cryptoKeyPath
-     *
-     * @return string The formatted crypto_key_path resource.
-     * @deprecated Use cryptoKeyName instead
-     */
-    public static function cryptoKeyPathName($project, $location, $keyRing, $cryptoKeyPath)
```

Comment:
This was removed because of a bug I fixed in https://github.com/googleapis/google-cloud-php/pull/5076, so this will need to be regenerated